eng/Version.Details.xml

```diff
@@ -17,9 +17,9 @@
       <Uri>https://dev.azure.com/devdiv/DevDiv/_git/vs-code-coverage</Uri>
       <Sha>06325faa0d16bac193af8d51026f03208034880d</Sha>
     </Dependency>
-    <Dependency Name="MSTest.TestFramework" Version="3.10.0-preview.25311.1">
+    <Dependency Name="MSTest.TestFramework" Version="3.10.0-preview.25366.1">
```

Comment:
I was expecting to also see an update here:

https://github.com/microsoft/testfx/blob/799f0b2cdcbef1577306748d9565b6117180cd49/eng/Versions.props#L12-L13

@nohwnd Do you know why it didn't update Versions.props?